sample-applications/luthier-sample-app/pom.xml

```diff
@@ -0,0 +1,136 @@
+        <checkstyle.config.location>../../checkstyle-style.xml</checkstyle.config.location>
+        <checkstyle.suppressions.location>../../checkstyle-suppressions.xml</checkstyle.suppressions.location>
+        <skipLuthierTests>true</skipLuthierTests>
+        <version.fili>0.12-SNAPSHOT</version.fili>
+    </properties>
+
+    <profiles>
+        <profile>
+            <!--      run tests and generate-test-resources if Lua is installed to /usr/local/bin/lua      -->
+            <id>lua-installed-local-bin</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <file>
+                    <exists>/usr/local/bin/lua</exists>
+                </file>
+            </activation>
+            <build>
+                <plugins>
+                </plugins>
+            </build>
+            <properties>
+                <skipLuthierTests>false</skipLuthierTests>
+            </properties>
+        </profile>
+        <profile>
+            <!--      run tests and generate-test-resources if Lua is installed to /usr/bin/lua      -->
+            <id>lua-installed-bin</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <file>
+                    <exists>/usr/bin/lua</exists>
+                </file>
+            </activation>
+            <build>
+                <plugins>
+                </plugins>
+            </build>
+            <properties>
+                <skipLuthierTests>false</skipLuthierTests>
+            </properties>
+        </profile>
+    </profiles>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>com.yahoo.fili</groupId>
+            <artifactId>luthier</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.yahoo.fili</groupId>
+            <artifactId>sample-applications</artifactId>
+            <version>${version.fili}</version>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.1</version>
+                <executions>
+                    <execution>
+                        <id>run-luthier-main</id>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <mainClass>com.yahoo.bard.webservice.application.LuthierMain</mainClass>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <!--
```

Comment:
Uncomment?

Reply:
This is no longer the launch mechanism.  I'm removing it.